components/NumberDisplay.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
```

Comment:
`import React from 'react';` isn't required in Next.js

```suggestion
import { FC } from 'react';
```